lib/semian/net_http.rb

```diff
@@ -87,14 +88,24 @@ def disabled?
       raw_semian_options.nil?
     end
 
+    def fatal_server_errors?
+      raw_semian_options.fetch(:fatal_server_errors)
```

Comment:
Where can you document this option? I'd suggest adding a section for adapter-specific configuration in the `Adapters` section.